src/Database/Adapter/Mongo.php

```diff
@@ -253,6 +253,28 @@ public function listCollections(): array
         return $list;
     }
 
+     /**
+     * Get Collection Size
+     * @param string $collection
+     * @return int
+     * @throws DatabaseException
+     */
+     public function getSizeOfCollection(string $collection): int
+    {
+         $namespace = $this->getNamespace();
+         $command = [
+             'collStats' => $namespace . '_' . $this->filter($collection),
+             'scale' => 1 
+         ];
+        try{
+            $result = $this->getClient()->query($command);
+        }
+        catch(Exception $e){
+            throw new DatabaseException("Failed to get collection size" . $e->getMessage());
```

Comment:
Let's use single quotes if we're not using string interpolation as it is faster, and also make sure our error messages are consistent across adapters

```suggestion
            throw new DatabaseException('Failed to get collection size: ' . $e->getMessage());
```